vectors_test.go

```diff
@@ -27,6 +27,8 @@ type TestVector struct {
 	Footer              string
 	ExpectFail          bool   `json:"expect-fail"`
 	ImplicitAssertation string `json:"implicit-assertion"`
+	Paserk              string
+	Comment             string
```

Comment:
Rather than adding onto the PASETO vectors test here, I think probably we can just create a separate set of tests for PASREK (since most of the PASETO fields are going to be missing from parsed PASREK data).